tools/dragen-transcriptome/4.0.3/dragen-transcriptome__4.0.3.cwl

```diff
@@ -0,0 +1,232 @@
+cwlVersion: v1.1
+class: CommandLineTool
+
+# Extensions
+$namespaces:
+    s: https://schema.org/
+    ilmn-tes: http://platform.illumina.com/rdf/ica/
+$schemas:
+  - https://schema.org/version/latest/schemaorg-current-http.rdf
+
+# Metadata
+s:author:
+    class: s:Person
+    s:name: Sehrish Kanwal
+    s:email: sehrish.kanwal@umccr.org
+
+# ID/Docs
+id: dragen-transcriptome--4.0.3
+label: dragen-transcriptome v(4.0.3)
+doc: |
+    Documentation for dragen-transcriptome v4.0.3
+
+# ILMN V1 Resources Guide: https://illumina.gitbook.io/ica-v1/analysis/a-taskexecution#type-and-size
+# ILMN V2 Resources Guide: https://help.ica.illumina.com/project/p-flow/f-pipelines#compute-types
+# ILMN Resources Guide: https://illumina.gitbook.io/ica-v1/analysis/a-taskexecution#type-and-size
+hints:
+    ResourceRequirement:
+        ilmn-tes:resources:
+            tier: standard
+            type: fpga
+            size: medium
```

Comment:
From 
```yaml
hints:
    ResourceRequirement:
        ilmn-tes:resources:
            tier: standard
            type: fpga
            size: medium
```

To 

```yaml
hints:
    ResourceRequirement:
        ilmn-tes:resources:tier: standard
        ilmn-tes:resources:type: fpga
        ilmn-tes:resources:size: medium
```

Reply:
I actually initially used 

```
hints:
    ResourceRequirement:
        ilmn-tes:resources:tier: standard
        ilmn-tes:resources:type: fpga
        ilmn-tes:resources:size: medium
```

but this errored out in wfr.7fd3884c46dd4e95808bd25c6443ca53 with the hardware issue.

Reply:
Gah!

https://github.com/umccr-illumina/cwl-iap/issues/189#issuecomment-880913357

Needs to be 
```
hints:
    ResourceRequirement:
        ilmn-tes:resources/tier: standard
        ilmn-tes:resources/type: fpga
        ilmn-tes:resources/size: medium
```

Reply:
I'll update the rest of the tools we've just changed, my bad!

Reply:
Resolved in #200

Reply:
updated in https://github.com/umccr/cwl-ica/pull/199/commits/73eaae0c24c7c1bf067785ecc45e399434dab8f3